CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [ICDAR](https://rrc.cvc.uab.es/?ch=2) format support (<https://github.com/openvinotoolkit/cvat/pull/2866>)
 - Added switcher to maintain poylgon crop behaviour (<https://github.com/openvinotoolkit/cvat/pull/3021>
 - Filters and sorting options for job list, added tooltip for tasks filters (<https://github.com/openvinotoolkit/cvat/pull/3030>)
+- Documentation on mask annotation (<https://github.com/openvinotoolkit/cvat/pull/3044>)
```

Comment:
Hmm.. 1.3.0 was release yesterday, the next release is 1.4.0, not 1.3.0 alpha. 
Looks like need to update CHANGELOG a bit and add new UNRELEASED section